File: TypeScript/abc191_b/main.ts
async function readAllInput(): Promise<string[]> {
    const input = await new Response(Deno.stdin.readable).text();
    return input.trim().split('\n');
}

function parseLineToNumbers(line: string): number[] {
    return line.trim().split(' ').map(Number);
}

function solve(nums: number[], x: number): string {
    let ret = nums.filter(a => a != x).join(" ");
    return ret;
}

const line = await readAllInput();
const nAndX = parseLineToNumbers(line[0]);
const _n = nAndX[0];
const x = nAndX[1];
const a = parseLineToNumbers(line[1]);

const result = solve(a, x);
console.log(result);

export{};

File: Python/abc191_b/main.py
def solve(lst, x):
    return list(filter(lambda a: a != x, lst))


def get_input():
    return input()


def get_input_int():
    return int(input())


def get_int_list():
    return list(map(int, input().split()))


def join_space_int_list(lst):
    return " ".join(map(str, lst))


def main():
    list = get_int_list()
    _n = list[0]
    x = list[1]

    a = get_int_list()

    result = join_space_int_list(solve(a, x))
    print(result)


if __name__ == "__main__":
    main()
